model/authorizations/v1/terms_review_response_type.model

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright (c) 2020 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Representation of an terms review response.
+struct TermsReviewResponse {
+	// Account ID of requesting user.
+	AccountId string
+
+	// Indicates which Organization the user belongs to.
+	OrganizationID string
+
+	// Defines whether Terms Avalable for the Organization of which terms are being reviewed.
```

Comment:
s/Avalable/Available/

Could the comment clarify what this means?
Is this true whenever RedirectUrl is set?